CHANGELOG.md

```diff
@@ -2,6 +2,28 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.22.0] - 2018-02-24
+### Changed
+- Update versions for:
+    - `react-dates` to `"16.3.2"`
+    - `react-dropzone` to `"4.2.8"`
+    - `react-markdown` to `"3.2.1"`
+    - `react-select` to `"1.2.1"`
```

Comment:
Since we're bumping major versions in these libraries, I bet that this will introduce some breaking changes in the component's CSS. In case any users are overriding this CSS, then their overrides won't work anymore. So, I think we should make a breaking change version bump to `1.0.0` (according to semver)

Reply:
And let's leave a note in the changelog that mentions that CSS stylesheets may be different

Reply:
I haven't actually looked into whether or not the CSS or markup has changed in these versions, I'm just guessing that it has.

Reply:
OK, will update to `1.0.0`.